pkg/cluster/delete.go

```diff
@@ -448,10 +448,11 @@ func (m *manager) Delete(ctx context.Context) error {
 		m.log.Printf("deleting OIDC configuration")
 		blobContainerURL := oidcbuilder.GenerateBlobContainerURL(m.env)
 		azBlobClient, err := m.rpBlob.GetAZBlobClient(blobContainerURL, &azblob.ClientOptions{})
+		OIDCIssuerPath := m.subscriptionDoc.Subscription.Properties.TenantID
 		if err != nil {
 			return err
 		}
-		err = oidcbuilder.DeleteOidcFolder(ctx, env.OIDCBlobDirectoryPrefix+m.doc.ID, azBlobClient)
+		err = oidcbuilder.DeleteOidcFolder(ctx, fmt.Sprintf("%s/%s", OIDCIssuerPath, m.doc.ID), azBlobClient)
```

Comment:
Since we use `fmt.Sprintf("%s/%s", OIDCIssuerPath, m.doc.ID)` in multiple places, write a function in `oidcbuilder.go` to call this. 

Something like:
```
func GetBlobName(tenantID string, docID string) string {
   return fmt.Sprintf("%s/%s", tenantID, docID)
}
```